zstacklib/zstacklib/utils/qemu.py

```
@@ -48,10 +50,20 @@ def get_version():
 
     return version
 
+def get_install_date():
+    r, o, e = bash.bash_roe("rpm -q --queryformat '%{INSTALLTIME}' qemu-kvm")
```

Ruff (0.14.4)

Unpacked variable `e` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)